scripts/link-them.sh

```diff
@@ -2,6 +2,7 @@
 # Copied from https://github.com/ef4/ember-auto-import/blob/master/scripts/link-them.sh
 set -e
 
+ln -s ../../../.. bin 2> /dev/null || true
```

Comment:
This gets called whenever a developer runs `yarn`, `yarn install` or `yarn prepare`